algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -455,9 +455,9 @@ case class BFItem[A](item: A, hashes: BFHash[A], width: Int) extends BF[A] {
 
   def ++(other: BF[A]): BF[A] =
     other match {
-      case bf @ BFZero(_, _)            => this
-      case bf @ BFItem(otherItem, _, _) => toSparse + otherItem
-      case _                            => other + item
+      case _ @BFZero(_, _)            => this
```

Comment:
instead of `_ @ BFZero(_, _)` can we write `BFZero(_, _)`? Similar below for BFItem?

Reply:
🤦‍♂ yeah totally 👍